pkg/oidcprovider/oidcprovider.go

```diff
@@ -192,7 +198,7 @@ func (p *provider) CreateAccessAndRefreshTokens(ctx context.Context, request op.
 // CreateAccessToken implements op.Storage.
 func (p *provider) CreateAccessToken(ctx context.Context, request op.TokenRequest) (accessTokenID string, expiration time.Time, err error) {
 	slog.Warn("oidcprovider.*service.CreateAccessToken", "request", request)
-	return strings.Join(request.GetScopes(), " "), time.Now().Add(time.Minute), nil
+	return strings.Join(request.GetScopes(), " "), time.Now().Add(time.Hour * 24), nil
```

Comment:
Is this for some jwt thing?

Reply:
I thought this set the lifetime of the jwt token, but I was wrong. Fixed